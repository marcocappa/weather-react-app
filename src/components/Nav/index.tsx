import React from 'react';
import { Link } from 'react-router-dom';

const Nav = (): JSX.Element => (
    <nav>
        <Link to="/">Home</Link> <Link to="/forecast">Forecast</Link>
    </nav>
);

export default Nav;
