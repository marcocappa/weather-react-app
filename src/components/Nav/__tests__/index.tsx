import React from 'react';
import { shallow } from 'enzyme';
import Nav from '../index';

describe('<Nav />', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(<Nav />);
        expect(wrapper).toMatchSnapshot();
    });
});
