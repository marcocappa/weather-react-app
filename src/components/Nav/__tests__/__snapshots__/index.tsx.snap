// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> should match snapshot 1`] = `
<nav>
  <Link
    to="/"
  >
    Home
  </Link>
   
  <Link
    to="/forecast"
  >
    Forecast
  </Link>
</nav>
`;
